src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/gui/ToggleButton.java

```diff
@@ -15,7 +15,7 @@
     private final ItemStack trueTexture;
     private final ItemStack falseTexture;
     private final ToggleListener callback;
-    boolean value;
+    public boolean value;
```

Comment:
Why would this be public and not private?

Reply:
i thought sonarcloud said it like that but will change and see if i get errors